includes/Api/ApiPricingPlan.php

```diff
@@ -55,6 +58,7 @@ public function change_plan() {
 			WUBTITLE_ENDPOINT . 'stripe/customer/update',
 			array(
 				'method'  => 'POST',
+				'timeout' => 13,
```

Comment:
why the timeout is 13?

Reply:
because the lambda function has a 12 second timeout, I added a second for safety

Reply:
put 15 seconds